inference-engine/src/gna_plugin/optimizer/gna_pass_manager.hpp

```diff
@@ -123,6 +123,14 @@ DECL_PASS(ReorderMaxPool);
  */
 DECL_PASS(HandleMultipleActivationsForTheLayer);
 
+/**
+ * @brief GNA doesn't provide intermediate results (sums) when the layer is fused with activation.
+ * When more layers use the sums as inputs (beside the activation) then the diagonal layer
+ * is inserted before the activation to forbid the fusing and make the sums exposed.
+ * This is observed in the multiple_activations_onGNA_INT16 test.
+ */
+DECL_PASS(ForbidActivationFusing);
+
 /**
  * @brief copy layer insertion required in cases where input layer does not have output memory
  */
```

Comment:
Please add tests or unblock some if there are already existing ones for this issue.

Reply:
The test is in different/internal repo: git@gitlab-icv.inn.intel.com:inference-engine/dldt-tests-closed.git